assembly/src/ast/module.rs

```diff
@@ -161,11 +161,11 @@ impl ModuleAst {
         match &self.docs {
             Some(docs) => {
                 assert!(docs.len() <= u16::MAX as usize, "docs too long");
```

Comment:
Maybe we should change asserts like this to some `WARN` or even `ERROR` logging message instead of panic?